test/test_core.py

```diff
@@ -5443,15 +5442,11 @@ def test_files(self):
 
     self.do_run_in_out_file_test('test_files.c')
 
-  def test_files_m(self):
-    # Test for Module.stdin etc.
-    # needs to flush stdio streams
-    self.set_setting('EXIT_RUNTIME')
```

Comment:
It sounds like this was testing this option before? Or am I misunderstanding the comment?

Reply:
Just adding a final newline is enough to flush the stdio streams.  No need for the explicit `EXIT_RUNTIME` here.  I don't think this pertains to the actual test, which is why I removed it.